pallets/liquidity-pools/src/message.rs

```diff
@@ -239,9 +239,6 @@ where
 		/// original `DecreaseRedeemOrder` message, i.e., the amount by which
 		/// the redeem order was actually decreased by.
 		tranche_tokens_payout: Balance,
-		/// The remaining amount of tranche tokens the investor still has locked
-		/// to redeem at a later epoch execution
-		remaining_redeem_order: Balance,
```

Comment:
NOTE: This was discussed, see [DAO slack comment](https://centrifugedao.slack.com/archives/C04HCK98KLL/p1692046938307889?thread_ts=1691760753.515679&cid=C04HCK98KLL).